src/controller/NoteController.ts

```diff
@@ -25,20 +25,17 @@ export class NoteController {
                         break;
                     case 'date':
 
-                        let first;
-                        let second;
-                        const dateOne = new Date(value[0]);
-                        const dateTwo = new Date(value[1]);
+                        if (Array.isArray(value)) {
+                            const dateOne = new Date(value[0]);
+                            const dateTwo = new Date(value[1]);
 
-                        if (dateOne < dateTwo) {
-                            first = startOfDay(dateOne);
-                            second = endOfDay(dateTwo);
+                            const first = (dateOne < dateTwo) ? startOfDay(dateOne) : startOfDay(dateTwo);
+                            const second = (dateOne < dateTwo) ? startOfDay(dateTwo) : startOfDay(dateOne);
```

Comment:
i think you mean `endOfDay`?
 and what do you think of formatting them to be smtm like 
```
startOfDay(dateOne < dateTwo ? dateOne : dateTwo);
```
up to you for whichever you think is more readable tho

Reply:
oh whoops great catch. I've never used ternary statements before so not sure which syntax is better. I'll go with yours haha

Reply:
we would still want to keep startOfDay and endOfDay though right?

Reply:
oh wait im dumb i see now

Reply:
i also just realized this but i was looking into possibly a better way to compare the dates and found min/max
https://date-fns.org/v2.19.0/docs/max
https://date-fns.org/v2.19.0/docs/min
so we wouldn't need ternary oops sorry for not bringing it up earlier